contracts/Escrow.sol

```diff
@@ -263,13 +263,13 @@ contract Escrow is ChainlinkClient, ConfirmedOwner, Context, AccessControlled
     * @dev This function ignores the stored Oracle contract address and
     * will instead send the request to the address specified
     * @param _oracle The Oracle contract address to send the request to
-    * @param _jobId The bytes32 JobID to be executed
+    * @param _jobId The bytes32 JobID to be executed (??)
```

Comment:
i think this is a chainlink thing but i'm not sure if it's proper to have it as a param? i think this might be something the oracle returns as basically a callback identifier sort of thingy

Reply:
The jobId is a chainlink thing, which we will have the proper value of once our EA is fully set up. You need to have it before calling the oracle, so we need to have it as a param